app/views/admin/tenants/index.html.erb

```diff
@@ -1,21 +1,72 @@
 <div id="tenants_header">
   <h1>Tenants</h1>
 </div>
+<div id="unapproved">
+  <h2>Unapproved Tenants</h2>
+  <table style="width:100%">
+    <tr>
+      <th>Organization</th>
+      <th>Location</th>
+      <th>Slug</th>
```

Comment:
I would delete this(and all instances of slugs in this view). As an admin, I don't care about slugs, but you could have each Organization link to the corresponding tenant's page.